handlers/jaeger.go

```diff
@@ -29,65 +29,19 @@ func ProxyJaeger(w http.ResponseWriter, r *http.Request) {
 	proxy.ServeHTTP(w, r)
 }
 
-/** End solution */
-
-// GetjaegerInfo provides the jaeger URLo, first by checking if a config exists
-// then (if not) by inspecting the Kubernetes Jaeger service in namespace istio-system
+// GetJaegerInfo provides the proxy Jaeger URL
 func GetJaegerInfo(w http.ResponseWriter, r *http.Request) {
 	conf := config.Get()
-	info, code, err := getJaegerInfo(getOpenshiftRouteURL, getService)
+	urlKiali, err := getOpenshiftRouteURL(conf.IstioNamespace, conf.KialiService)
```

Comment:
The point is that before I was getting the url from getOpenshiftRouteURL too, in kubernetes should work because it's using the kubernetes client

Reply:
+1
Assuming the same host is correct. It is basically same functional behaviour that today but without using the openshift route. It's a valid assumption that will use the same host (and scheme).

Will refactor it.

Reply:
@aljesusg the client is indeed the same, but it doesn't *know* the Openshift API it doesn't even know what a _Route_ object is, so it would probably return an error or something empty.

Reply:
aaa I understand thanks @jotak